collect_app/src/main/res/layout/select_minimal_widget_answer.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:paddingHorizontal="@dimen/margin_standard"
+    android:paddingVertical="@dimen/margin_small">
+
+    <com.google.android.material.textfield.TextInputEditText
+        android:id="@+id/choices_search_box"
```

Comment:
Nit: I got really confused about this name because I thought it was related to the `search`/`autocomplete` appearance. This represents the answer, right? Or the selection?

Reply:
This is that `spinner` button which contains answers inside.

Reply:
I would just call it something like `answer` but `spinner_button` feels like a big improvement, too.

Reply:
Done.